src/EnergyPlus/WindTurbine.cc

```diff
@@ -779,18 +771,14 @@ namespace WindTurbine {
 
         static char const TabChr('\t'); // Tab character
 
-        int mon;           // loop counter
-        bool wsStatFound;  // logical noting that wind stats were found
-        bool warningShown; // true if the <365 warning has already been shown
```

Comment:
[src/EnergyPlus/WindTurbine.cc:782]:(style),[variableScope],The scope of the variable 'mon' can be reduced.
[src/EnergyPlus/WindTurbine.cc:783]:(style),[variableScope],The scope of the variable 'wsStatFound' can be reduced.
[src/EnergyPlus/WindTurbine.cc:784]:(style),[variableScope],The scope of the variable 'warningShown' can be reduced.